src/Servers/Kestrel/Transport.Quic/src/Internal/QuicConnectionContext.cs

```diff
@@ -55,15 +55,8 @@ public override async ValueTask DisposeAsync()
         {
             try
             {
-                if (_closeTask != default)
```

Comment:
This condition is backwards right? Did this change fix a known bug?

Reply:
It looked that way to me.  I don't think this quic transport has gotten a lot of mileage yet.